thundra/integrations/requests.py

```diff
@@ -14,7 +14,8 @@ def __init__(self):
     def get_operation_name(self, wrapped, instance, args, kwargs):
         prepared_request = args[0]
         url = prepared_request.url
-        return url
+        parsed_url = urlparse(url)
```

Comment:
if url parse fails, operation name should be `prepared_request.url` by default. So there should be catch for `urlparse(url)`

Reply:
Added exception check for it in new commit, please check